spec/Section 2 -- Language.md

```diff
@@ -178,6 +169,16 @@ significant way, for example a {StringValue} may contain white space characters.
 No {Ignored} may appear *within* a {Token}, for example no white space
 characters are permitted between the characters defining a {FloatValue}.
 
+**Byte order mark**
+
+UnicodeBOM :: "Byte Order Mark (U+FEFF)"
+
+The *Byte Order Mark* is a special Unicode code point which may appear at the
+beginning of a file which programs may use to determine the fact that the text
+stream is Unicode, and what specific encoding has been used.
+
+As files are often concatenated, a *Byte Order Mark* may appear anywhere within
```

Comment:
I don't have enough background to judge that. How often and in which ares is that used these days that we need to consider that?

Reply:
UTF-8 doesn't need a BOM, so since that's become the dominant encoding this comment is a little anachronistic (BOM is common in other encodings like UTF-16), but I think still important for completeness.

This same clause appears in the JavaScript spec for similar reasons.

Reply:
I should point out that this is an editorial change. The current spec has this same clause, it was just in an awkward location before we had a space to discuss Unicode support directly.

Reply:
Yeah ... GraphQL Java actually already handles that :-)